### DISCUSSION TITLE
HTTPS protocol - 404 page - Docker RunTipi

### DISCUSSION THREAD

**Discussion by Lafabriquemanolo** [runtipi/runtipi-appstore#4074] [Q&A] [answered]
@CiliaFred 
Hello, I'm having a problem installing the Invoice Ninja app via Tipi.
I'm new to server management, but in general everything works perfectly when I expose an app via a subdomain configured as "A" in DNS and pointing to my server.
However, the Invoice Ninja app seems to work only with the http protocol (e.g. http://invoice.mydomain.fr:8881), but not to the expected https://invoice.mydomain.fr. 
Port 8881 is open, as are ports 80 and 443.
I've already tried a number of things: reboot, reinstall the app, change the subdomain for exposure, check the settings in app.env... in short, I'm drying up and it's a problem that it only works in http, because I want to connect it to Stripe to collect certain invoices.
If you have an idea, or need more info, don't hesitate.
Thanks

**Comment by CiliaFred** [comment]
Hello,

Normally, you mustn't access the application thought port 8881.
This port is used by Traefik (part of runtipi) to map your port 443 to port 8881.

https://invoice.mydomain.fr ⇾ DNS ⇾ Runtipi Server (443) ⇾ Traefik (443) ⇾ Docker network (8881) ⇾ Invoice Ninja Container (80) 

You must control that the following parameter is correctly set during the installation.
![image](https://github.com/runtipi/runtipi-appstore/assets/1827520/4695d792-7181-4fdc-86a8-da823b7f8118)

**Comment by Lafabriquemanolo** [comment]
Hello @CiliaFred,

Thank you very much for your feedback. I don't know why I can't find a solution with this installation. The reverse proxy works effectively for all the apps I've installed, but for some reason I get a 404 page when I try to use https://client.chedmatic.fr/.

I've got a DNS A entry pointing to the client, I've set up the reverse proxy, but it doesn't open Invoice Ninja unless I use http://client.chedmatic.fr:8881.

If you ever have a solution or detect an anomaly in my settings, I'd love to hear from you. If not, too bad for me, I've been waiting for Invoice Ninja to arrive for a while, but I'll organize things differently!

PS: I see that your tipi interface is in French, you can reply in French if you are too.

![Capture d’écran 2024-08-19 100648](https://github.com/user-attachments/assets/e5fe43ec-e56b-4bab-9aed-de9746620ff8)

**Reply by CiliaFred** [reply, marked as answer]
Hello,

I just try with the last version of runtipi, and i've got the same issue.....

There is probably an issue with the traefik configuration. I will try to fix the issue soon.

**Reply by CiliaFred** [reply]
Fixed here : [https://github.com/runtipi/runtipi-appstore/pull/4482](url)

Just need to wait for the validation.

**Comment by OriginalSimon** [comment]
When using https I also get 404 page not found error with Nextcloud.

edit: rm Crafty

**Comment by Lafabriquemanolo** [comment]
Super thanks a lot @CiliaFred the update went through, it works perfectly.

@OriginalSimon concerning nexcloud it's not normal, but concerning Crafty, I think it doesn't work in https (it's indicated in the doc)

**Reply by OriginalSimon** [reply]
![зображення](https://github.com/user-attachments/assets/2274f729-c7cd-4a41-9ccf-9147c3fb8429)
![зображення](https://github.com/user-attachments/assets/e753198c-4d7d-490f-8d3b-cabe00521fb7)
![зображення](https://github.com/user-attachments/assets/eb84feda-c1bf-4f9a-b18c-3c081f391291)
